### DISCUSSION TITLE
Security Hub API

### DISCUSSION THREAD

**Discussion by niraj-ranjan-comprinno** [aws/aws-sdk-java-v2#3887] [General]
I'm trying to figure out how to get the Security Score from a set of enabled controls (CIS) via CLI/Boto/Golang, etc. I don't see any options where to pull it from or how to calculate it.

**Comment by debora-ito** [comment]
Hey @niraj-ranjan-comprinno you should reach out to the specific SDK team in their own Github repos.

As a reminder, the SDK teams don't have expertise in the subject of each and every one of the AWS Services, so if this information is not in the Security Hub documentation, we'll need to reach out internally to the service team, and to get a response will take a while.

**Reply by niraj-ranjan-comprinno** [reply]
HI @debora-ito ,
You can suggest me for AWS JAVA SDK V2.
